code/obj/machinery/door/airlock.dm

```diff
@@ -857,44 +833,41 @@ var/global/list/cycling_airlocks = list()
 		if (!(src in cycling_airlocks[src.cycle_id]))
 			cycling_airlocks[src.cycle_id] += src
 
-/obj/machinery/door/airlock/open()
-	if (!src.density || src.welded || src.locked || src.operating == 1 || (!src.arePowerSystemsOn()) || (src.status & NOPOWER) || src.isWireCut(AIRLOCK_WIRE_OPEN_DOOR))
-		return 0
-	src.use_power(OPEN_CLOSE_POWER_USAGE)
-	if (src.linked_forcefield)
-		power_usage += LINKED_FORCEFIELD_POWER_USAGE
-	.= ..()
-
-	playsound(src.loc, src.sound_airlock, 25, 1)
-
-	if (src.cycle_id)
-		for (var/obj/machinery/door/airlock/D in cycling_airlocks[src.cycle_id])
-			// if they share entry id, don't close, e.g. double doors facing space.
-			if (src.cycle_enter_id && src.cycle_enter_id == D.cycle_enter_id)
-				continue
-			if (D.operating) //can happen with really short airlocks, see atlas south maint
-				SPAWN(0.5 SECONDS)
+/obj/machinery/door/airlock/open(var/manual_actuation = FALSE)
+	if (!manual_actuation)
+		if (!src.density || src.welded || src.locked || src.operating == 1 || (!src.arePowerSystemsOn()) || (src.status & NOPOWER) || src.isWireCut(AIRLOCK_WIRE_OPEN_DOOR))
+			return 0
+	if(!(src.status & NOPOWER))
+		src.use_power(OPEN_CLOSE_POWER_USAGE)
+	. = ..()
+	if(!manual_actuation)
+		playsound(src.loc, src.sound_airlock, 25, 1)
+
+		if (src.cycle_id)
+			for (var/obj/machinery/door/airlock/D in cycling_airlocks[src.cycle_id])
+				// if they share entry id, don't close, e.g. double doors facing space.
+				if (src.cycle_enter_id && src.cycle_enter_id == D.cycle_enter_id)
+					continue
+				if (D.operating) //can happen with really short airlocks, see atlas south maint
+					SPAWN(0.5 SECONDS)
+						D.close()
+				else
 					D.close()
-			else
-				D.close()
-
-/obj/machinery/door/airlock/close()
-	//split into two sets of checks so failures to close due to lacking power will cause linked shields to deactivate
-	if ((!src.arePowerSystemsOn()) || (src.status & NOPOWER) || src.isWireCut(AIRLOCK_WIRE_OPEN_DOOR))
-		if (src.linked_forcefield)
-			src.linked_forcefield.setactive(0)
-			power_usage -= LINKED_FORCEFIELD_POWER_USAGE
-		return
```

Comment:
Removing this early return makes it possible to close doors by hand without power.

Reply:
oops

Reply:
fixed in cbe5b1f